plex_trakt_sync/plex_api.py

```diff
@@ -180,29 +208,64 @@ def audio_channels(self):
         """
         Set to 1.0, 2.0, 2.1, 3.0, 3.1, 4.0, 4.1, 5.0, 5.1, 5.1.2, 5.1.4, 6.1, 7.1, 7.1.2, 7.1.4, 9.1, or 10.1
```

Comment:
I don't see how your new logic would ever return a value like `7.1.2`.

Please remove unreachable items from the list.